src/LoadingSpinner.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 
 const LoadingSpinner = () => (
-  <img className="loading-spinner" alt="Loading..." src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/items/poke-ball.png" />
+  <img className="loading-spinner" alt="Loading,,," src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/items/poke-ball.png" />
```

Comment:
We should break a more serious case in a test, for example I had an idea that we could break the fetch in the test like change the url at `const { data: pokemonList, error, isLoading } = useApi('https://pokeapi.co/api/v2/pokemon/?limit=784', mapResults)` to make it invalid.

Reply:
@RikuLa I'm afraid that kind of thing might be too tricky to figure out, considering they're not familiar with the API and maybe with Open APIs at all. I was thinking we should keep the breakages simple so it's more GHA exercise and less debugging exercise.

Reply:
I'm not sure if it would be that tricky. I think it would be nice to have a very real issue that a failing test prevents from "being committed to master". We can break it in a super obvious way like instead of saying `.../pokemon` we can put `.../digimon` or such.

Reply:
I've changed this to an actual display error (previous pokemon instead of next in the "Next" link)

Reply:
Yeah, thats obvious as well and a mistake one _could_ make in reality :)